deploy/binary-build-and-deploy-scripts/install-driver.sh

```diff
@@ -32,12 +33,12 @@ cat /root/.ssh/id_rsa.pub >> /host/root/.ssh/authorized_keys
 chmod 700 /host/root/.ssh/
 chmod 600 /host/root/.ssh/authorized_keys
 
+touch /host/etc/ssh/sshd_config
```

Comment:
My colleague suggested to add `mkdir -p /host/etc/ssh/` before the `touch` command in case some cluster doesn't have that ssh directory. I checked in the minimal code that should work and leave it up to you whether to add this addition or not.